cmd/docker-app/bundle_test.go

```diff
@@ -9,46 +9,16 @@ import (
 
 func TestMakeInvocationImage(t *testing.T) {
 	testcases := []struct {
-		name      string
-		imageName string
-		namespace string
-		meta      metadata.AppMetadata
-		expected  string
-		err       string
+		name     string
+		meta     metadata.AppMetadata
+		expected string
+		err      string
 	}{
-		{
-			name:      "specify-image-name",
```

Comment:
Don't remove this one, it is still valid in case the user defines his own invocation image name.

Reply:
Hmmm... But as @simonferquel said in https://github.com/docker/app/pull/454#discussion_r256325757 this PR also removes this possibility

Reply:
Ok right 👍